TopQuarkAnalysis/TopEventProducers/src/StEvtSolutionMaker.cc

```diff
@@ -2,17 +2,55 @@
 //
 
 #include <memory>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <fstream>
 
-#include "TopQuarkAnalysis/TopEventProducers/interface/StEvtSolutionMaker.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "AnalysisDataFormats/TopObjects/interface/StEvtSolution.h"
+#include "TopQuarkAnalysis/TopKinFitter/interface/StKinFitter.h"
+//#include "TopQuarkAnalysis/TopJetCombination/interface/TtJetCombinationProbability.h"
```

Comment:
The commented out included file doesn't even exist: this line can be removed

Reply:
Thanks, removed.